README.md

```diff
@@ -26,7 +26,7 @@ Documentation of ART: https://adversarial-robustness-toolbox.readthedocs.io
 Get started with [examples](examples/README.md) and [tutorials](notebooks/README.md)
 
 The library is under continuous development. Feedback, bug reports and contributions are very welcome. 
-Get in touch with us on [Slack](https://ibm-art.slack.com) (invite [here](https://join.slack.com/t/ibm-art/shared_invite/enQtMzkyOTkyODE4NzM4LTlkMWY3MzgyZDA4ZDdiNzUzY2NhMjc5YmFhZTYzZGYwNDM4YTE1ODhhNDYyNmFlMGFjNWY4ODgyM2EwYTFjYTc))!
+Get in touch with us on [Slack](https://ibm-art.slack.com) (invite [here](https://join.slack.com/t/ibm-art/shared_invite/enQtMzkyOTkyODE4NzM4LTA4NGQ1OTMxMzFmY2Q1MzE1NWI2MmEzN2FjNGNjOGVlODVkZDE0MjA1NTA4OGVkMjVkNmQ4MTY1NmMyOGM5YTg))!
```

Comment:
Why is this link being changed?

Reply:
This change is from my commit on branch `master` where I made this change to update a stale link to our Slack channel invitation. This PR is based on branch `master`